extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosInterface.java

```diff
@@ -134,7 +134,7 @@ MediaItemSearchResponse listMediaItems(Optional<String> albumId, Optional<String
     if (pageToken.isPresent()) {
       params.put(TOKEN_KEY, pageToken.get());
     }
-    HttpContent content = new JsonHttpContent(new JacksonFactory(), params);
+    HttpContent content = new JsonHttpContent(GsonFactory.getDefaultInstance(), params);
```

Comment:
I wonder if we can just use `this.jsonFactory` instead of creating a new instance here and below on L171?

Reply:
That does seem like a better approach overall.

Reply:
Done.